src/template.json

```diff
@@ -1,49 +1,41 @@
 {
   "builders": [
     {
-      "access_key": "{{user `aws_access_key`}}",
       "ami_groups": "{{user `AmiGroups`}}",
       "ami_name": "{{user `AmiImageName`}}",
-      "assume_role": {
-        "role_arn": "{{user `aws_role_arn`}}"
-      },
-      "encrypt_boot": "{{user `EncryptBoot`}}",
+      "communicator": "winrm",
       "force_delete_snapshot": "true",
+      "force_deregister": true,
+      "imds_support": "v2.0",
       "instance_type": "{{user `InstanceType`}}",
-      "launch_block_device_mappings": [
-        {
-          "delete_on_termination": true,
-          "device_name": "/dev/sda1",
-          "volume_size": "{{user `VolumeSize`}}",
-          "volume_type": "gp3"
-        }
-      ],
-      "profile": "{{user `aws_profile`}}",
-      "region": "{{user `aws_region`}}",
-      "secret_key": "{{user `aws_secret_key`}}",
-      "shutdown_behavior": "terminate",
       "snapshot_groups": "{{user `SnapshotGroups`}}",
       "source_ami_filter": {
         "filters": {
-          "name": "ubuntu/images/*ubuntu-jammy-22.04-amd64-server-*",
+          "architecture": "x86_64",
+          "name": "Windows_Server-2022-English-Full-Base*",
           "root-device-type": "ebs",
           "virtualization-type": "hvm"
         },
         "most_recent": true,
         "owners": [
-          "099720109477"
+          "amazon"
         ]
       },
-      "ssh_username": "{{user `SshUsername`}}",
       "tags": {
         "BuiltBy": "Packer",
+        "Department": "{{user `Department`}}",
```

Comment:
why are `Department` and `Project` tags added?  i thought we are only using `CostCenter` tag now?

Reply:
whoops, I should've removed those after copying the template from https://github.com/Sage-Bionetworks-IT/packer-base-winserver2019/blob/v0.0.1/src/template.json